<ui version="4.0" >
 <class>profileAssignmentPage</class>
 <widget class="QWidget" name="profileAssignmentPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>177</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>Profiles</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4" >
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>When AC Adaptor is plugged in</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="KComboBox" name="acProfile" >
        <property name="maximumSize" >
         <size>
          <width>280</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>When on battery Power</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="KComboBox" name="batteryProfile" >
        <property name="maximumSize" >
         <size>
          <width>280</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>When battery is at low level</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="KComboBox" name="lowProfile" >
        <property name="maximumSize" >
         <size>
          <width>280</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>When battery is at warning level</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" >
       <widget class="KComboBox" name="warningProfile" >
        <property name="maximumSize" >
         <size>
          <width>280</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
